config/config.go

```diff
@@ -140,3 +141,8 @@ type SyslogMetrics struct {
 	Network  string
 	Addr     string
 }
+
+type Pprof struct {
```

Comment:
__[Golint]__

exported type Pprof should have comment or be unexported


<!-- Comment by [SideCI](https://sideci.com) -->